### DISCUSSION TITLE
Molina - ToC

### DISCUSSION THREAD

**Discussion by sudhasridhar** [CMSgov/price-transparency-guide#659] [General]
Good Morning - Question on toc file published by Molina. There are about 667 in_network files in the toc; per cms guideline on the toc file https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/table-of-contents, all 667 rate files are pointing to all of their 5 HIOS plans. 

Is anyone else using their toc and interpreting differently? when there is a question about payer published toc files, do payers have contact details for follow-up?

**Comment by shaselton-usds** [comment]
It looks like you're correct. They definitely could improve their file, but as it stands, it is still valid and you can pull all the information you want out of it. In this case, it seems like just pulling the in network file locations and mapping them to the 5 HIOS plans.

**Reply by bryanbriegel** [reply]
@shaselton-usds - what are the thoughts of use of a 5 byte HIOS issuer ID, when a 10 byte HIOS ID is available?

**Comment by sudhasridhar** [comment]
Thank you, this is exactly how we are interpreting the file. It seems like most plans are provider network focused and plan agnostic (ie) for a provider network, the negotiated rates are the same irrespective of the plan metal level.
